unstructured/ingest/v2/processes/connectors/onedrive.py

```diff
@@ -89,10 +89,10 @@ class OnedriveIndexer(Indexer):
 
     def list_objects(self, folder, recursive) -> list["DriveItem"]:
         drive_items = folder.children.get().execute_query()
-        files = [d for d in drive_items if d.is_file]
+        files = [d for d in drive_items if d.file is not None]
         if not recursive:
             return files
-        folders = [d for d in drive_items if d.is_folder]
+        folders = [d for d in drive_items if d.is_folder is not None]
```

Comment:
@rbiseck3 
Changed to match [this example](https://github.com/vgrem/Office365-REST-Python-Client?tab=readme-ov-file#example-download-the-contents-of-a-driveitemfolder-facet), but can revert it.